submissions/Vlada-khom/html-css-popup/index.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/css/styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <title>Popup</title>
+  </head>
+  <body>
+    <header class="header">
+      <nav>
+        <ul>
+          <li>
+            <a href="#" class="navLink">Mail</a>
+          </li>
+          <li>
+            <a href="#" class="navLink">Images</a>
+          </li>
+          <li class="burger">
+            <input type="checkbox" id="burgerCheckbox" class="input" />
+            <label for="burgerCheckbox" class="burgerLabel navLink">
+              <img
+                class="imgLink burgerImg"
+                src="./img/popup-button.png"
+                alt="menu"
```

Comment:
Read carefully [this article](https://htmlacademy.ru/blog/articles/alt-text) to understand what you should do with all `alt` attributes in you page)